code/modules/projectiles/guns/projectile/automatic/vintorez.dm

```diff
@@ -23,7 +23,7 @@
 	silenced = TRUE
 	init_firemodes = list(
 		SEMI_AUTO_NODELAY,
-		FULL_AUTO_400
+		BURST_2_ROUND
```

Comment:
one_hand_penalty is unchanged even though full auto was removed, also a 2 round burst is very low for a rifle.

Reply:
I can put it up to three if you like. Not sure why one hand penalty matters in this case, it's still a rifle.

Reply:
one_hand_pently.

5 for SMGs, 10 for bullpup rifles, 15 for FA rifles, 20 for "full-sized" rifles

Fairly unclear and meaningless, I don't see a problem with taking artistic freedom here. One-handing them is going to be unviable eitherways.

Reply:
So should I put the burst up to 3? I kind of was hoping the vintorez would start to shift to a role similar to the kovac as a sort of battle rifle, especially given it has a scope. That's why I gave it a 2 round burst. The AK would naturally become the main assault rifle of excelsiors due to higher bursts and FA.